src/test/resources/UserEditInformations.feature

```diff
@@ -0,0 +1,49 @@
+Feature: 
+  As a user
+  I want to be able to change my username and password
+  
+  Background: 
+    Given database is empty
+    And user with id 1234 exists and has username "user", password "secure_password" and refreshToken "token"
+    
+  Scenario: Successful change of username
+    When user requests change of username with value "kangaroo" and refreshToken "token" and id "1234"
+    Then response contains key "new-username" and value "kangaroo"
+    And response status code is 201
+
+  Scenario: Successful change of password
+    When user requests change of password with value "pig-system" and refreshToken "token" and id "1234"
+    Then response contains key "new-password" and value "****"
```

Comment:
Naa The response should not return anything in my oppinion.
It can return something like `status: ok and message: password successfully changed.`